### DISCUSSION TITLE
Mount a storageclass with a minio external storage server

### DISCUSSION THREAD

**Discussion by dborgi** [okd-project/okd#1690] [General]
I have to create services in jboss and I need to save the logs, install the Red Hat OpenShift Logging operator, and I want to configure a storageclass with a minio server that is external to my cluster, do you have any guide to configure it? Thanks

**Comment by vrutkovs** [comment]
Minio provides a block storage, not a file storage, so it cannot be mounted as, say, NFS. There are however projects which may emulate that - see https://github.com/yandex-cloud/k8s-csi-s3 for example

**Reply by dborgi** [reply]
thanks for the reply.

My okd is onpremise and my storage is also going to be onpremise
 What can be a production recommended nfs variant for logging ?
